# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/patches/detect-system-db4.7.diff
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- config_office/configure.in	2008-05-20 16:29:33.000000000 +0000
+++ config_office/configure.in.new	2008-07-07 17:35:57.000000000 +0000
@@ -3407,7 +3407,7 @@
         ], []
     )
     AC_MSG_CHECKING([whether db is at least 4.1])
-    for v in 1 2 3 4 5 6; do
+    for v in 1 2 3 4 5 6 7; do
        AC_TRY_RUN([
 #include <db.h>
 
